reactos/reactos
# USETUP: Simplify partition UI management

## Purpose

The purpose of this PR is two-fold:
1. Code reduction and simplification when using the partition helpers in the user interface, because similar code will also be used in the 1st-stage GUI setup;
2. Simplification in view of supporting GPT disks (for UEFI platforms).

Advice for reviewing: check each individual commit.

Some strings will require re-translation!

## How to test?

1. Once you land in the disk/partition page, try creating/deleting partitions (primary, extended, logical) and verify that things behave correctly (logical partitions can only be created inside the extended partition, etc.)
2. Once you've chosen a partition where to install ReactOS, verify that the screens for choosing a type of formatting correctly display the partition name and disk;
3. When the actual formatting and check-disk takes place (different screens), verify that the partition name and disk are correctly displayed.
4. Finally, verify that ReactOS is correctly installed!
